backend/src/tasks/helpers/saveTrustymailResultsToDb.ts

```diff
@@ -0,0 +1,29 @@
+import { domain } from 'process';
+import { connectToDatabase, Domain } from '../../models';
+import * as fs from 'fs';
+
+// TODO: Find a way to only send one query to the db
+export default async (path: string): Promise<void> => {
+  const domainId = path.split('/')[path.split('/').length - 1].split('.')[0];
+  const jsonData = await fs.promises.readFile(path);
+  await connectToDatabase();
+  await Domain.createQueryBuilder()
+    .update(domain)
+    .set({
+      trustymailResults: () => `'${jsonData}'`
+    })
+    .where('id = :id', { id: domainId })
+    .execute();
+  Domain.createQueryBuilder()
+    .update(domain)
+    .set({
+      trustymailResults: () =>
+        `jsonb_set("trustymailResults"::jsonb, '{0,Timestamp}', to_jsonb(current_timestamp::timestamp), true)`
```

Comment:
What does current_timestamp::timestamp do?

Reply:
current_timestamp is a postgres timestamp that includes date, time, and timezone (GMT in this case.). ::timestamp recasts this to different format which removes the timezone.

Reply:
Timestamp is now in json object so this line is gone.